galley/pkg/config/analysis/analyzers/util/hosts.go

```diff
@@ -52,6 +52,12 @@ func GetScopedFqdnHostname(scope, namespace, host string) ScopedFqdn {
 	return ScopedFqdn(scope + "/" + name)
 }
 
+// GetScopedNamespaceAndName splits ScopedFqdn back to scope namespace and host
+func (s ScopedFqdn) GetScopedNamespaceAndName() (string, string) {
+	parts := strings.Split(string(s), "/")
+	return parts[0], parts[1]
+}
+
```

Comment:
Move this in the file to be adjacent to the ScopedFqdn type definition. 

We should also probably move GetScopedFqdnHostname and rename it to NewScopedFqdn, to be as clear as possible.